### PR TITLE
[#1665] rename frontend component

### PR DESCRIPTION
**Pull request by LuoZhijie-tom**
Add changes according to [#1665]

# Proposed commit message
```
The components are all named with a prefix `v-`. However, most reserved
syntax used by Vue uses the `v-` prefix.

Let's change the `-v` prefix to prefix `c-`, so that it will be less
likely to cause confusion and conflicts.
```

# Other information
I tried to just remove the prefix previously, but the `summary` is a reserved word in html, thus the `v-summary` can't be renamed. So I add `-c` as a prefix.

### CODE REVIEW

**Comment by LuoZhijie-tom** [comment]
Hi @Zhou-Jiahao-1998 , I have solved the conflict and now created a new pull request

**Comment by Zhou-Jiahao-1998** [comment]
@LuoZhijie-tom Hi there, please update the cypress tests as well to reflect the changes. You can find the files under `frontend/crypress` folder.

**Comment by LuoZhijie-tom** [comment]
> @LuoZhijie-tom Hi there, please update the cypress tests as well to reflect the changes. You can find the files under `frontend/crypress` folder.

Hi @Zhou-Jiahao-1998 , I have modified the code and it passed the cypress test now, could you have a look?

File: frontend/.eslintrc.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "@vue/typescript"
   ],
   "rules": {
+    "vue/multi-word-component-names": "off",
```
Comment (Zhou-Jiahao-1998) [inline frontend/.eslintrc.json:11]: May I know what is this for and why is it needed for this PR?

**Review by Zhou-Jiahao-1998 — APPROVED** [review]
LGTM

File: frontend/.eslintrc.json (hunk 1 above)
Comment (LuoZhijie-tom) [reply frontend/.eslintrc.json]: Oh sorry I forgot to remove this, it is to allow the naming of one-word component like "ramp", it was added previously when I try to remove the "-v", but later when I added the "-c", this setting is useless.  I have removed it now.

Comment (Zhou-Jiahao-1998) [reply frontend/.eslintrc.json]: Thank you for the clarification!

**Comment by damithc** [comment]
Guys, shall we push this PR forward?

**Review by dcshzj — CHANGES_REQUESTED** [review]
Need to update [all the diagrams](https://docs-1836-pr-reposense-reposense.surge.sh/dg/report.html) as well.

**Comment by LuoZhijie-tom** [comment]
> Need to update [all the diagrams](https://docs-1836-pr-reposense-reposense.surge.sh/dg/report.html) as well.

OK noted!

**Comment by LuoZhijie-tom** [comment]
@dcshzj I have updated the diagrams, could you have a look?

**Review by dcshzj — APPROVED** [review]
LGTM!

**Comment by damithc** [comment]
Guys, shall we merge this?

**Pull request merged**